plugins/optimization-detective/types.ts

```diff
@@ -49,8 +49,18 @@ export type OnLCPWithAttributionFunction = typeof onLCPWithAttribution;
 export type OnINPWithAttributionFunction = typeof onINPWithAttribution;
 export type OnCLSWithAttributionFunction = typeof onCLSWithAttribution;
 
+export type LogFunction = ( ...message: any[] ) => void;
+
+export interface Logger {
+	prefix: string;
+	log: LogFunction;
```

Comment:
Let's add `info` too.

```suggestion
	log: LogFunction;
	info: LogFunction;
```

Reply:
Done in 0d1ea45.